website_authorized_service_centers/demo/brands_and_partners_demo.xml

```diff
@@ -0,0 +1,196 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@josemoralesp all the demo data created here must be created as legacy data, I brought it from the actual yoytec website, this info was not in the database.